docs: mention that apiKey provided to maintainer can be safely share (#51)
algolia/docsearch-website: docs/config-file.md

  },
```

> _Note that you can use [advanced synonym thanks to Algolia][5]. Our scraper only

Reviewer: ```suggestion
_Note that you can use [advanced synonym with Algolia][5]. Our scraper only
```
